scripts/index.js

```diff
@@ -0,0 +1,126 @@
+const state={
+    color:"teal",
+    temp:71,
+    garden:"☘️🌻🌵💐",
+    
+};
+
+const increatTemp = ()=>{
+    const tempContainer=document.getElementById("temp")
+    state.temp +=1;
+    tempContainer.textContent=`${state.temp}`;
+    
+    updateColorofTemp(state.temp);
+    updateGarden(state.temp);
+};
+
+const decreatTemp = ()=>{
+    const tempContainer=document.getElementById("temp")
+    state.temp +=1;
```

Comment:
Ooops!

```suggestion
    state.temp -=1;
```

Reply:
Thank you for your feedback, Chris ! I have refactored my code follow your commits.